buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -21,7 +21,7 @@ class AirbytePythonPlugin implements Plugin<Project> {
             installVirtualenv = true
             pip 'flake8:3.8.4'
             pip 'black:20.8b1'
-            pip 'mypy:0.790'
+            pip 'mypy:0.812'
```

Comment:
bump gradle mypy to latest version so running something like `gradle format` doesn't overwrite with an older mypy version